pages/roadmap.js

```diff
@@ -139,10 +140,23 @@ export default function RoadmapPage() {
                 items={roadmapData.outcomes.future}
               />
             </div>
+
+            <p className="mt-12 mb-4 mx-auto text-center text-gray-800 lg:w-1/2">
+              If you want to know more about the format of this roadmap, watch this recording from one of our SIG meetings:
+            </p>
+
+            <div className="mx-auto lg:w-1/2">
+              <YouTubeEmbed
+                id="u83V2gIUGHU"
+                appendSrc="?start=86"
+              />
+            </div>
+
             <Warning
               className="lg:w-1/2 mt-8 mx-auto"
               title="Warning for Contributors"
-              description="This road map reflects the priorities of the core team. If you want to contribute a feature that's not a priority, feel free to let us know on the corresponding Github issue so we can discuss what's the best way to proceed and implement it yourself." />
+              description="This road map reflects the priorities of the AsyncAPI Initiative. If you want to contribute a feature that's not a priority, feel free to let us know on the corresponding Github issue so we can discuss what's the best way to proceed and implement it yourself."
```

Comment:
```suggestion
              description="This road map reflects the priorities of the AsyncAPI Initiative. If you want to contribute a feature from the roadmap, feel free to let us know on the corresponding Github issue so we can discuss what's the best way to proceed and implement it yourself."
```

for me it should be open on any roadmap item, even the ones with the highest priority. GitHub issue is where further scope is discussed and possible engagement